build.js

```diff
@@ -0,0 +1,49 @@
+const { execSync } = require('child_process');
```

Comment:
Note to future self: this JavaScript logic should not be introduced, it's already written in Python in the xeus-python addon.

I should split the xeus-python addon module into two modules: the logic for building and packing an emscripten env **AND** the addon itself.